scripts/deploy.js

```diff
@@ -0,0 +1,11 @@
+const { execSync } = require('child_process');
+const args = require('minimist')(process.argv.slice(4));
+
+const { folder, distId, bucketName, packageName, tagName } = args;
+const bucketPath = `${bucketName}/${packageName}/${tagName}`;
+
+// Sync to s3 and invalidate
+const syncCommand = `aws s3 sync . s3://${bucketPath} --delete --acl public-read`
```

Comment:
I think you'll need to install the `aws` CLI if it does not exist. If won't be available on the runners by default.

Reply:
Done!